package.json

```diff
@@ -1459,6 +1459,10 @@
               "xamlTools": {
                 "description": "%configuration.dotnet.server.componentPaths.xamlTools%",
                 "type": "string"
+              },
+              "roslynCopilot": {
+                "description": "%configuration.dotnet.server.componentPaths.roslynCopilot%",
```

Comment:
this needs a corresponding string in package.nls.json
https://github.com/dotnet/vscode-csharp/blob/main/package.nls.json#L32